scheduler.go

```diff
@@ -8,31 +8,30 @@ import (
 // Scheduler struct, the only data member is the list of jobs.
 // - implements the sort.Interface{} for sorting jobs, by the time nextRun
 type Scheduler struct {
-	jobs [MAXJOBNUM]*Job // Array store jobs
-	size int             // Size of jobs which jobs holding.
-	loc  *time.Location  // Location to use when scheduling jobs with specified times
+	jobs []*Job         // Array store jobs
```

Comment:
Probably don't need this comment any more
```suggestion
	jobs []*Job
```